### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by TJ09** [facebook/fbt#395]
## 💬 Questions and Help

#372 was resolved back in September, but it is still not easily possible to use FBT with React 18 since there has not been a new release since v1.0.0 last April. I'm not sure what the state of the main branch is, but a new point release would be nice.

Alternatively I'm willing to backport #374 to the v1.0.0 branch to enable a v1.0.1.

**Comment by richardDobron** [comment]
You can now specify an overrides property in your package.json to override and enforce nested dependency versions.

```diff
{
  "dependencies": {
    // ...
    "fbt": "^1.0.0",
    "react": "^18.2.0"
    // ...
-  }
+  },
+  "overrides": {
+    "fbt@^1.0.0": {
+      "react": "^18.2.0"
+    }
+  }
}

```

**Comment by voideanvalue** [comment]
Closing as I published v1.0.2 to npm today: https://www.npmjs.com/package/fbt
